lib/liquid/tags/raw.rb

```diff
@@ -3,7 +3,7 @@
 module Liquid
   class Raw < Block
     Syntax = /\A\s*\z/
-    FullTokenPossiblyInvalid = /\A(.*)#{TagStart}\s*(\w+)\s*(.*)?#{TagEnd}\z/om
+    EndRawTag = /\A(.*)#{TagStart}\s*endraw.*?#{TagEnd}\z/om
```

Comment:
We have tags that inherit from Liquid::Raw, so will need this to work with other delimiters if we do need to support garbage characters after the delimiter name